### DISCUSSION TITLE
Pesan Error Nomor Telp

### DISCUSSION THREAD

**Discussion by imamsudarajat04** [imamsudarajat04/nongkrongkoding#42] [General]
![image](https://user-images.githubusercontent.com/70099731/208019956-b5a632c4-3913-4dd2-9f2d-dacd93ed39b1.png)

pesan errornya kesamping nih @falfikri

**Comment by falfikri** [comment]
ooh itu, kalo ga salah belum aku kasih class nya tu nanti aku kasih

**Reply by imamsudarajat04** [reply]
oke nanti tunggu aku kelar bikin update profilenya
